specs/DateTimePicker-Visual-Updates-Spec.md

```diff
@@ -0,0 +1,235 @@
+| ReplacementColor | Gets or sets the Color to use instead of the non-transparent pixels of the SourceElement.
+
+
+# API Details
+
+```csharp
+namespace Microsoft.UI.Xaml.Controls
+{
+
+[MUX_PROPERTY_CHANGED_CALLBACK(TRUE)]
+[MUX_PROPERTY_CHANGED_CALLBACK_METHODNAME("OnPropertyChanged")]
+unsealed runtimeclass ElementBackgroundGrid : Windows.UI.Xaml.Controls.Grid
+{
+    ElementBackgroundGrid();
+
+    Windows.UI.Xaml.UIElement SourceElement { get; set; };
+    Windows.UI.Color ReplacementColor { get; set; };
+
+    static Windows.UI.Xaml.DependencyProperty SourceElementProperty{ get; };
+    static Windows.UI.Xaml.DependencyProperty ReplacementColorProperty{ get; };
+}
+
+}
+```
+
+
+# Appendix
+<!-- Anything else that you want to write down for posterity, but 
+that isn't necessary to understand the purpose and usage of the API.
+For example, implementation details. -->
+
+## DatePicker
+
+In WinUI 2.2, controls began to be re-designed to have rounded corners, marking a shift in our overall design language. 
+There's more information on rounded corners in WinUI [here](https://docs.microsoft.com/en-us/windows/uwp/design/style/rounded-corner), 
+but the general purpose of the rounded corners design shift is to evoke warmth and trust, 
+and make the UI easier for users to visually process. 
+Since this shift, certain controls have adopted the new styling and recieved rounded corners, but some have not. 
+This creates a strong visual inconsistency in WinUI apps, where certain pieces of an app may look modern and others may look dated. 
+Examples of this include the DatePicker and TimePicker controls. While the corners of the controls themselves are rounded, the selection rectangle within the control is still squared off. Buttons and other areas inside these controls are also still squared off, making them look dated when placed alongside other controls that are more modernly designed, such as 
+[NavigationView](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/navigationview).
+
+Recently, [ListView was given design updates](https://github.com/microsoft/microsoft-ui-xaml-specs/blob/user/anawish/ListViewGridViewDesignUpdates/active/ListViewGridView/listviewgridview-designupdates.md) to round the corners of its items. DatePicker and TimePicker should follow in these footsteps, and make sure that they adopt any of the ListView design elements that are relevant. This includes general rounding of selection visuals, as well as rounded grey backplates for items that get hovered. 
+
+This spec will detail a number of changes to update the designs of DatePicker and TimePicker so that they are more 
+visually aligned with modern WinUI controls. At a high level, some key parts of these updated styles are: a rounded selection rectangle; rounded backplates on items, buttons, and scroll buttons on  hover; and new animations that are aligned with other controls. 
```

Comment:
We'll have to address the input-eating problem before this will be viable.

Reply:
Added an explanation of turning off hit-testing to a couple of places in the text.